### ISSUE TITLE
Features Requests 

### ISSUE THREAD

**Issue by idelafue** [zendesklabs/linkets#11]
1. Is there any way for the tickets to get linked both ways when you link the first one? Currently you have to link the first ticket to the second and then you have to link the second ticket to the first. This extra step has our agents not doing it both directions.

2. Is there any way to link tickets without searching for them in your search box? We'd like to be able to manually link one ticket to another using ID or something similar https://www.screencast.com/... The issue for us is having hundreds of thousands of tickets (eventually millions) and the search function is not always the fastest way to find a ticket when we already have the ticket we want to link open in another tab.

3. Would it be possible to build the functionality of creating a linked ticket from the widget. We have the "linked ticket" app installed for Zendesk and it's really nice to be able to click one button to create a ticket that you know you want linked. Instead of having to create a new ticket and then search in that new ticket for the old ticket to link to the new ticket and vice versa. https://www.screencast.com/...

4. Is there any API access to Linkets and creating linked tickets so we could do this outside of Zendesk (we use the standard zendesk API to create and update tickets from Magento).

**Comment by tlambert03** [comment]
just stumbled onto this feature request when looking for a way to interact with linkets from the API... just wanted to point out: since linkets are ultimately just ticket tags, you can roll your own solution with the existing zendesk API using the following tag syntax:
```
blocked by: 'lnkt:blckdby:ID'
blocks: 'lnkt:blcks:ID'
is chlid: 'lnkt:chld:ID'
is parent: 'lnkt:prnt:ID'
relates to: 'lnkt:rlts:ID'
```
where `ID` is the ID of the other ticket...  For example to make ticket 1 be blocked by ticket 2, add the tag:

```
{
    'id': 1,
    'tags': ['lnkt:blckdby:2'],
}
```
